setup.py

```diff
@@ -26,7 +26,7 @@
     'renku==2.2.0',
     'astroquery @ git+https://github.com/oda-hub/astroquery#egg=astroquery',
     'aqsconverters @ git+https://github.com/oda-hub/aqsmodel-converters@simple-autolog#egg=aqsconverters',
-    'nb2workflow @ git+https://github.com/oda-hub/nb2workflow@fix-jinja2-version#egg=nb2workflow',
+    'nb2workflow',
```

Comment:
it's better to set `>=...` once it's bumped

Reply:
Was it umpoed after https://github.com/oda-hub/nb2workflow/pull/92 ?

Reply:
I upgraded the version but will it bump automatically?

Reply:
I am not sure we understand word "bump" in the same way here. Bump means to me increment version. Anyway you seem to have added the constrain so it's enough.

Reply:
yes that is what I meant, VERSION increment, thanks